packages/internal-playwright-helpers/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "ES2017",
+    "module": "commonjs",
```

Comment:
I think we can make this ESM only or SSNext

Reply:
Does this address your suggestion: https://github.com/inrupt/typescript-sdk-tools/pull/78/commits/0f99267beb45c4720b6b2b81f52b424ccaebd465 ?

Reply:
I think so, yes.